generators/entity-client/templates/vue/src/test/javascript/spec/app/entities/entity-management.component.spec.ts.ejs

```diff
@@ -96,6 +96,7 @@ describe('Component Tests', () => {
     it('should load a page', async () => {
       // GIVEN
       <%= entityInstance %>ServiceStub.retrieve.resolves({ headers: {}, data: [{ id: <%- tsKeyId %> }] });
+      comp.previousPage = null;
```

Comment:
Could you remove this line and change the `// WHEN` part with `comp.loadPage(2);`, it would be more like reality

Reply:
Done!